data/saas/config/mandrill_config.yml

```diff
@@ -0,0 +1,44 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Mandril SaaS Config
+  type: mandrill
+  description: A sample schema representing the Mandril connector for Fided
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: api_key
+
+  client_config:
+    protocol: https
+    host: <domain>
+
+  consent_requests:
+    opt_in:
+      - method: POST
+        path: /allowlists/add
+        body: |
+          {
+            "key": "<api_key>",
+            "email": "<email>"
+          }
+        data_uses: [advertising.first_party]
+    opt_out:
+      - method: POST
+        path: /allowlists/delete
+        body: |
+          {
+            "key": "<api_key>",
+            "email": "<email>"
+          }
+        data_uses: [ advertising.first_party]
```

Comment:
Are data uses hierarchical? For example, if advertising is deemed not executable, would advertising.first_party automatically not being executable? 

(That is not how my POC treats it, I assume they are separate data uses)

Reply:
Yes, they are. We should assume that, for a "policy"-type matching rule, any child nodes of a parent node in a `fideslang` taxonomy should match

Reply:
yikes, what does that mean if someone says they want to opt-out of `advertising` but opt-in to `advertising.first_party`, that's a conflict. luckily this first version won't be looking at data uses while we evaluate this further.